.github/workflows/clp-core-build.yaml

```diff
@@ -64,6 +64,13 @@ jobs:
         run: tar -xvf clp_binaries.tar
         working-directory: ${{github.workspace}}/binaries
 
+      - name: Log in to the Github Packages Container registry
+        uses: docker/login-action@v1
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+          
```

Comment:
Commenting here because github won't let me comment on lines not part of the diff:
* Can we change the name for the artifact to `clp-binaries-focal.tar`? That way it should be published as `clp-binaries-focal.tar.zip`
* Unnecessary newline before `build-bionic`